e2e/embedded.spec.ts

```diff
@@ -160,6 +160,65 @@ describe('Embedded works', () => {
     });
   });
 
+  it('Update books within author', async () => {
+    const books = author.books.map((book: any) => {
+      return { ...book, name: `${book.name}-edit` };
+    });
+    const { updateAuthorBooks } = await server.makeSuccessRequest({
+      query: `
+        mutation updateAuthorBooks($authorId: ID!, $input: [UpdateBookInput!]!) {
+          updateAuthorBooks(authorId: $authorId, input: $input) {
+            id
+            name
+          }
+        }
+      `,
+      variables: {
+        authorId: author.id,
+        input: books,
+      },
+    });
+    expect(updateAuthorBooks).toEqual(books);
+  });
+
+  it('Update books within author: return error if parent not found', async () => {
+    const response = await server.makeFailRequest({
+      query: `
+        mutation updateAuthorBooks($authorId: ID!, $input: [UpdateBookInput!]!) {
+          updateAuthorBooks(authorId: $authorId, input: $input) {
+            id
+            name
+          }
+        }
+      `,
+      variables: {
+        authorId: '5e6b4b5b1c9d440000d2c7f3',
```

Comment:
Use `000000000000000000000000` as it's more descriptive
Or small refactor to make this value reusable with a name
ex:

```ts
// test-helper.ts
const NOT_FOUND_ID = '000000000000000000000000'
```